src/main.cpp

```diff
@@ -57,7 +57,21 @@ int main()
 		std::cout << "Rolling with advantage: " << utils::RollWithAdvantage() << std::endl;
 		std::cout << "Rolling with disadvantage: " << utils::RollWithDisadvantage()
 				  << std::endl;
-		Battle();
+		Creature goblin(11, 14, 35, 16, 14, 10, 8, 7, 12, "Goblin");
```

Comment:
I believe it's time we start allocating the different creatures in the heap memory. Please turn this into a heap allocation, and treat it accordingly